src/plugins/data_views/server/index_patterns_api_client.ts

```diff
@@ -65,4 +65,9 @@ export class IndexPatternsApiServer implements IDataViewsApiClient {
       soClient: this.savedObjectsClient,
     });
   }
+
+  async getExistingIndices(indices: string[]) {
```

Comment:
hm. so to get the method onto the `data_views_api_client`, I had to add `getExistingIndices` to the `IDataViewsApiClient`. Since `index_patterns_api_client` also extends `IDataViewsApiClient`, we need to define this here. but does this ever get called? confused about what this is for 🤔

Reply:
Looking at the current state of the code, I wouldn't write it this way if I was writing it over. In fact, I might be tempted to factor out the DataViewsApiClient - it allows for public and server implementation BUT currently expects them to be the same. We don't have a use for that here. Its a layer of code thats not really serving a purpose.

The other way to go would be to implement `getExistingIndices` on `src/plugins/data_views/public/data_views_service_public.ts` - the public specific DataView code. The `http` dependency would need to be added to `src/plugins/data_views/public/data_views_service_public.ts:18` but that shouldn't be too bad.